style.css

```diff
@@ -0,0 +1,131 @@
+*,
+*::after,
+*::before {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  background-color: #e5e5e5;
+  margin-bottom: 20px;
+}
+
```

Comment:
- [ ] [REQUIRED] Good job so far! Please give correct style(`font-famy, font size, font-weight`) to all `texts` 🙏🏼 

`[HINT]`
```css
/* ----------Logo ------------- */
font-family: 'Poppins';
font-style: normal;
font-weight: 700;
font-size: 18px;

/* ----------Primary Text: h1 ------------- */
font-family: 'Poppins';
font-style: normal;
font-weight: 700;
font-size: 40px;

/* ----------Paragraph Text: p tag ------------- */
font-family: 'Poppins';
font-style: normal;
font-weight: 400;
font-size: 16px;

/* ----------Let's Connect: Text ------------- */
font-family: 'Poppins';
font-style: normal;
font-weight: 500;
font-size: 16px;
```

![image](https://github.com/Akorede3133/portfolio/assets/63915024/0852059c-bc53-4b82-a7a2-e74342acf97f)